integration_versions.md

```diff
@@ -0,0 +1,54 @@
+# Integrations
```

Comment:
Can we add a markdown comment to the top of the file saying it was generated from `.github/scripts/generate_table_versions.rb`?